docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+six
```

Comment:
You can add the docs requirements directly in the `docs` section of `setup.py` - https://github.com/inspirehep/inspire-next/blob/master/setup.py#L94

Then you can mention in the docs that they can be installed with `pip install -e .[docs]`